contracts/interfaces/dao/IDao.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity ^0.8.4;
+
+interface IDao {
+  struct ProposalMetaData {
+    address creator;
+    string info;
+    mapping(address => bool) voters;
```

Comment:
This could be where we store voting power (which is V_mul * N_tokens) per user address.

Reply:
Two issues with that.
1) Voting power changes incredibly often. Having transfer update every single proposal's voting weight on every single transfer would be suboptimal.
2) The IDao doesn't have scope on what's being used to vote; in this case, the Asset contract's ERC20. That's why it solely tracks voters. The Asset contract will iterate over and calculate vote count from token count and vote weight as needed.